Add baseTemplateRepoHash on metadata.json

andreriesco opened the pull request.

Commit:
metadata: Add baseTemplateRepoHash on metadata.json

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>

Commit:
project-updater.xsh: Open merge window for .doc README.md

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>

microhobby commented on scripts/create-from-template.xsh:
```diff
@@ -297,13 +297,17 @@ if not os.path.exists(f"{new_project_path}/.gitlab-ci.yml"):
 # template name
 # container name
 # base TOR used when created
+base_template_repo_hash_file = open(f"{template_folder}/../.git/ORIG_HEAD", "r")
+base_template_repo_hash = base_template_repo_hash_file.read().replace("\n", "")
```
@andreriesco take a look at the Copilot suggestions

Commit:
python3Console: Focus on debug console (use it on local debug)

Use debug console also for local debugging, to standardize with remote
debugging.

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>

Commit:
assets:Add Icon-Template.pptx,to make new templates contributions easier

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>

Commit:
project-updater.xsh: Fix

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>
